typegate/native/src/runtimes/s3.rs

```diff
@@ -16,6 +19,22 @@ struct S3Client {
     endpoint: String,
 }
 
+fn create_bucket(client: S3Client, bucket: &str) -> Result<Bucket, S3Error> {
```

Comment:
rename to get_bucket? the current name may indicate this indeed creates a bucket